diff --git a/etc/texi2pod.pl b/etc/texi2pod.pl
index cd0ffd9435f..1c753c5a8fb 100644
--- a/etc/texi2pod.pl
+++ b/etc/texi2pod.pl
@@ -311,7 +311,7 @@ while(<$inf>) {
 	@columns = ();
 	for $column (split (/\s*\@tab\s*/, $1)) {
 	    # @strong{...} is used a @headitem work-alike
-	    $column =~ s/^\@strong{(.*)}$/$1/;
+	    $column =~ s/^\@strong\{(.*)}$/$1/;
 	    push @columns, $column;
 	}
 	$_ = "\n=item ".join (" : ", @columns)."\n";
